prwkv/rwkvrnnmodel.py

```diff
@@ -262,6 +265,10 @@ def generate(self,
 
                 next_token = token_id
                 
+            if update_state:
```

Comment:
I understand what you mean by this ok hmmmm probably should have it as a property on the model rather than thrown into the generate function